extensions/resource-deployment/images/azure-sql-vm.svg

```diff
@@ -0,0 +1,25 @@
+<svg id="eaa44e3d-16b6-4034-8ab3-28e7277dbd71"  xmlns="http://www.w3.org/2000/svg"  width="18" height="18" viewBox="0 0 18 18">
```

Comment:
please sync up with Raden, the guideline for this page is we will use black-white themed icons.